src/UiElement/Registry.php

```diff
@@ -68,7 +68,7 @@ public function getUiElements(): array
     /**
      * @inheritdoc
      */
-    public function jsonSerialize()
+    public function jsonSerialize(): mixed
```

Comment:
In that case the PHP version in composer.json is wrong.

😭

Reply:
What can we can do ?